core/app/help.go

```diff
@@ -23,6 +23,45 @@ import (
 	"github.com/google/gapid/core/app/flags"
 )
 
+// MakeDocs
+func MakeDoc(ctx context.Context, name string) {
+	filename := name + ".md"
+	w, _ := os.Create(filename)
```

Comment:
This should just print to stdout so that it makes it easier to automate this and possibly wrap the generated contents in static content.

Reply:
Fixed